spline-shadow/client-ui/src/app/components/lineage/lineage-details/schema-details/schema-details.component.html

```diff
@@ -30,8 +30,25 @@
     <schema [schemaId]="'output_schema'" [schemaType]=1 [schema]="getOutput()"></schema>
   </div>
 </div>
-<div *ngIf="!detailsInfo">
-  <div class="details-title">
-    No details to show
+<div *ngIf="!detailsInfo && executedLogicalPlanVM">
```

Comment:
Maybe u could have```*ngIf="!detailsInfo``` on the higher level not having to repeat it.

Reply:
Yes we could but I've done that keeping in mind that it would be easier the day we want to put every single panel as an angular lib